src/services/upper-bar-manager/move-controls-manager.ts

```diff
@@ -0,0 +1,58 @@
+import { KalturaPlayer, Logger, ui } from '@playkit-js/kaltura-player-js';
+import { UpperBarManager } from './upper-bar-manager';
+import { IconDto } from './models/icon-dto';
+
+const { redux } = ui;
+
+export class MoveControlsManager {
+  private readonly player: KalturaPlayer;
+  private readonly logger: Logger;
+  private store: any;
+  private upperBarManager: UpperBarManager;
+  private currentState: any;
+  private iconIds: Map<string, number>;
+
+  constructor(player: KalturaPlayer, logger: Logger, upperBarManager: UpperBarManager) {
+    this.player = player;
+    this.logger = logger;
+    this.store = redux.useStore();
+    this.upperBarManager = upperBarManager;
+    this.store.subscribe(this.handleStoreChange.bind(this));
+    this.currentState = this.store.getState();
+    this.iconIds = new Map();
+  }
+
+  private get bottomBarRegistryManager(): any {
+    return (this.player.getService('bottomBarRegistryManager') as any) || undefined;
```

Comment:
i think our service names are usually uppercase

Reply:
mmm not from what i've seen (dualscreen, upperbarmanager, timeline, etc..)

Reply:
oh, you're right.
i think i was thinking of something else then. probably the class names.